docker/docker-compose.ci.yml

```
@@ -0,0 +1,43 @@
+version: "3.6"
+
+services:
+  app:
+    image: sirius-lpa-frontend:latest
+    build:
+      context: ..
+      dockerfile: ./docker/sirius-lpa-frontend/Dockerfile
+    depends_on:
+      - pact-stub
+    ports: ["8888:8888"]
+    environment:
+      PORT: 8888
+      SIRIUS_URL: http://pact-stub:8080
+      SIRIUS_PUBLIC_URL: http://localhost:8080
+
+  pact-stub:
+    build: ./pact-stub
+    ports: ["8080:8080"]
+    environment:
+      PACT_DIR: ./pacts
+      PORT: 8080
+    volumes:
+      - ../pacts:/app/pacts
+
+  puppeteer:
+    build: ./puppeteer
+    depends_on:
+      - app
+      - pact-stub
+    environment:
+      - LHCI_BUILD_CONTEXT__CURRENT_HASH=$CIRCLE_SHA1
```

We need to fix this in user-management too. 
```suggestion
      - LHCI_BUILD_CONTEXT__CURRENT_HASH=$GITHUB_SHA
```